android/src/main/java/com/swmansion/reanimated/layoutReanimation/ScreensHelper.java

```diff
@@ -0,0 +1,66 @@
+package com.swmansion.reanimated.layoutReanimation;
+
+import android.view.View;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ScreensHelper {
+
+  public static View getTabNavigator(View view) {
+    View currentView = view;
+    while (currentView != null) {
+      if (currentView.getClass().getSimpleName().equals("ScreenContainer")) {
+        return currentView;
+      }
+      if (currentView.getClass().getSimpleName().equals("Screen")
+          && currentView.getParent() != null
+          && currentView
+              .getParent()
```

Comment:
We call `.getParent()` twice, maybe we could store the result somewhere and break this condition into two separate conditionals?